exonum/src/node/consensus.rs

```diff
@@ -978,6 +986,11 @@ impl NodeHandler {
     /// Requests a block for the next height from all peers with a bigger height. Called when the
     /// node tries to catch up with other nodes' height.
     pub(crate) fn request_next_block(&mut self) {
+        if !self.is_enabled {
+            trace!("Do not send a request for the next block because the node is paused.",);
```

Comment:
Nit: Bogus comma; also "Not sending a request..." would make more sense grammatically.